backend/typescript/rest/entityRoutes.ts

```diff
@@ -0,0 +1,91 @@
+import express, { Router } from 'express';
+import { Entity } from "../models/entity.mgmodel"
+import EntityService from "../services/implementations/EntityService";
+import { EntityRequestDTO } from "../services/interfaces/IEntityService";
+
+export const entityRouter: Router = Router();
+
+/* Create entity Object */
+entityRouter.post("/", async (_req, res) => {
+    const entityObj: EntityRequestDTO = _req.body.entity;
```

Comment:
I think it's just `_req.body`

Also you can remove the underscore, it was previously in the test endpoint to denote that the `req` param isn't being used but that's not the case anymore